test/visual.html

```diff
@@ -163,6 +163,10 @@ <h3>Selection Tests</h3>
 
 <p>Even in IE&lt;9, the background color of the parens and square root radical should be the background color of the selection.
 
+<p>Static math with mouseEvents set to false should not interact with the mouse: <span class="static-math-no-mouse-events">12 + 34</span></p>
+
+<p>Even in the case where it has an empty element: <span class="static-math-no-mouse-events">\sqrt{} </span></p>
+

```

Comment:
Can you add an inert field with an embedded MathQuillMathField?  I'd like to have that in the visual check to make it easy to ensure that embedded fields are not broken during future changes.

Reply:
So, I added this locally and discovered that embedded MathQuillMathFields just don't work.  That's not a stopper from adding this functionality, since it's new and opt-in.  However, I think we should make effort to allow that long-term, so I'll add a new task for doing so.